### DISCUSSION TITLE
"docker run --privileged dind-rootless" dind-rootless host network cannot access to the services inside the container created by dind-rootless

### DISCUSSION THREAD

**Discussion by yushiqie** [moby/moby#48962] [Q&A]
Terminal 1:
Inside dind-rootless contianer, run a busybox container and listen on `:1234` 

![20241127111230](https://github.com/user-attachments/assets/ea32bb9c-378f-49c0-8db2-e99575c722ce)

Terminal 2:
Inside dind-rootless host network can not access to `172.17.0.2:1234`
![20241127111418](https://github.com/user-attachments/assets/91a828e9-5a8f-421d-9f72-38753d22ac34)

`How to configure vpnkit or slirp4netns so that the host can access container services`

**Comment by AkihiroSuda** [comment]
In rootless mode the container IP addresses are only accessible from containers.
Port forwarding (`docker run -p`) still works, of course.
